### PR TITLE
SRE1-566 Moved charts from k8s-email to shared

### PR DESCRIPTION
**Pull request by Mykolaichenko**

### CODE REVIEW

**Comment by white-hat** [comment]
@epicfilemcnulty @Mykolaichenko could you please write 2-3 lines of description what these charts are doing?
@zunkree please confirm that we are not duplicating functionality of other charts here, it seems suspicious that core team somehow survived without dockerhub secrets and log delivery for all this time

File: stable/fluentd-shipper/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      labels:
+        app: {{ .Chart.Name }}
+        release: {{ .Release.Name }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmaps.yaml") . |  sha256sum  }}
+    spec:
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+      imagePullSecrets:
+      - name: dockercfg
```
Comment (dmitriy-lukyanchikov) [inline stable/fluentd-shipper/templates/daemonset.yaml]: I think this needs to be in values so we can change what secret to use

**Comment by dmitriy-lukyanchikov** [comment]
also need to add to values default values like tolerations, nodeAffinity, and add possibility to define nodeSelector

in values this should look like this

```
nodeAffinity: {}
imagePullSecrets: []
nodeSelector: {}
tolerations:
- effect: NoSchedule
  key: node-role.kubernetes.io/master
```

**Comment by dmitriy-lukyanchikov** [comment]
Ithink also to not use or not to create additional secret with docekr secrets i propose to add optional secret if you decide to create needed docekrcfg secret in this chart like this

```
---
apiVersion: v1
kind: Secret
metadata:
  name: dockercfg
  labels:
    chart: {{ .Chart.Name }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
data:
  .dockercfg: '{{ (print "{\"https://index.docker.io/v1/\":{\"email\":\"use@example.com\",\"auth\":\"" .Values.dockerHubLogin "\"}}") | b64enc }}'
type: kubernetes.io/dockercfg
```

or something like this

**Comment by Mykolaichenko** [comment]
@dmitriy-lukyanchikov Ok guys I just need to move it to shared charts as-is, because I don't wanna to copy paste the same chart (or implement git submodule) which we are using for logging in different projects so it will be available over helm repo: 
https://github.com/AnchorFree/k8s-airflow/tree/master/additional_charts/fluentd-shipper
https://github.com/AnchorFree/k8s-email/tree/master/helm/system/fluentd-shipper
https://github.com/AnchorFree/k8s-shared/tree/master/fluentd-shipper
https://github.com/AnchorFree/k8s-rmq/tree/master/common/fluentd-shipper

I'm going to merge it as-is if there nothing sensitive inside (LGTM) and schedule improvements in second iteration. 

@dmitriy-lukyanchikov @white-hat Please notify me if you don't agreed with following approach

File: stable/fluentd-shipper/templates/daemonset.yaml (hunk 1 above)
Comment (Mykolaichenko) [reply stable/fluentd-shipper/templates/daemonset.yaml]: will be implemented in INFRA-8676

**Pull request merged**